src/utils/update/request.to.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { Request } from "express";
```

Comment:
i'd move the location of this file and also include the word 'mapper' in the file name (so it matches the other mapper naming pattern under /trusts)
As it is not purely update related I would create this file as
src/utils/request.to.entity.mapper.ts